en/appendices/4-0-upgrade-guide.rst

```diff
@@ -20,10 +20,10 @@ Now that you can see all the warnings, make sure these are fixed before proceedi
 Upgrade to PHP 7.2
```

Comment:
Isnt that header then also kind of wrong?

Reply:
Yea, it's confusing CakePHP 4.4 requires PHP 7.4 not 7.2, but this doc is about the upgrade which requires 8.0. It's a mess.

Reply:
It might be time to overhaul it to 7.4 then (cake4), and from there on to 8.1 (cake5)

Reply:
Yes, but this is for the 4.x migration notes only.

Reply:
Yes, oversight. It's changed in a new commit.